src/spec/category-results-service-spec.ts

```diff
@@ -47,7 +71,7 @@ describe('category results service', () => {
 		categories = createCategoriesData();
 	});
 
-	it('should generate the correct html', async (done: DoneFn) => {
+	xit('should generate the correct html', async (done: DoneFn) => {
```

Comment:
So is this going to be added back in?

Reply:
Yeah these are in progress. But not until I get a free moment from other work.